src/components/backend-ai-agent-list.ts

```diff
@@ -672,46 +705,44 @@ export default class BackendAIAgentList extends BackendAIPage {
+            <div class="layout horizontal center-justified flex progress-bar-section">
               <div class="layout horizontal start resource-indicator">
                 <mwc-icon class="fg green">developer_board</mwc-icon>
-                <span style="padding-left:5px;">${rowData.item.cpu_slots}</span>
+                <span class="monospace" style="padding-left:5px;">${rowData.item.used_cpu_slots}/${rowData.item.cpu_slots}</span>
                 <span class="indicator">${_t('general.cores')}</span>
               </div>
               <span class="flex"></span>
-              <lablup-progress-bar id="cpu-usage-bar" progress="${rowData.item.cpu_current_usage_ratio}"
-                                   buffer="${rowData.item.cpu_total_usage_ratio}"
-                                   description="${rowData.item.current_cpu_percent}%"></lablup-progress-bar>
+              <lablup-progress-bar id="cpu-usage-bar" progress="${rowData.item.cpu_total_usage_ratio}"
+                                   description="${rowData.item.total_cpu_percent}%"></lablup-progress-bar>
             </div>` : html``}
           ${rowData.item.mem_slots ? html`
-            <div class="layout horizontal center flex">
+            <div class="layout horizontal center-justified flex progress-bar-section">
               <div class="layout horizontal start resource-indicator">
                 <mwc-icon class="fg green">memory</mwc-icon>
-                <span style="padding-left:5px;">${rowData.item.mem_slots}</span>
-                <span class="indicator">GB</span>
+                <span class="monospace" style="padding-left:5px;">${rowData.item.used_mem_slots}/${rowData.item.mem_slots}</span>
+                <span class="indicator">GiB</span>
               </div>
               <span class="flex"></span>
-              <lablup-progress-bar id="mem-usage-bar" progress="${rowData.item.mem_current_usage_ratio}"
-                                   buffer="${rowData.item.mem_total_usage_ratio}"
-                                   description="${rowData.item.current_mem}GB"></lablup-progress-bar>
+              <lablup-progress-bar id="mem-usage-bar" progress="${rowData.item.mem_total_usage_ratio}"
+                                   description="${rowData.item.total_mem_percent}%"></lablup-progress-bar>
             </div>` : html``}
           ${rowData.item.cuda_gpu_slots ? html`
-            <div class="layout horizontal center flex">
+            <div class="layout horizontal center-justified flex progress-bar-section">
               <div class="layout horizontal start resource-indicator">
                 <img class="indicator-icon fg green" src="/resources/icons/file_type_cuda.svg"/>
-                <span style="padding-left:5px;">${rowData.item.cuda_gpu_slots}</span>
+                <span class="monospace" style="padding-left:5px;">${rowData.item.used_cuda_gpu_slots}/${rowData.item.cuda_gpu_slots}</span>
```

Comment:
@Sujin-Kim1  It's not as "identically" same as agent list in control-panel.
In WebUI for now in Allocation column, the admin user will see two information with progress bar:
- current allocated resources
- total amount of resources

in Utilization column, It will show up the ratio of actual resources usage in agent (it will contain other usages such as background processes in the agent(host))

And also, I don't know why there's hard-coded number `100 `in control-panel you mentioned.
Could you guess and tell me the reason?

Reply:
According to this [commit](https://github.com/lablup/backend.ai-control-panel/commit/89614f822c3a8ccbc18e9830955d5500a22bf569), capacity is delivered from the server as 0. So I guess @adrysn handled it as 100.

Reply:
Since getting an empty object as `live_stat` from Redis server has been solved, I think we shall leave this commit as is.